src/psd_tools/api/layers.py

```diff
@@ -561,6 +627,52 @@ def bbox(self):
             self._bbox = Group.extract_bbox(self)
         return self._bbox
 
+    def add_layer(self, layer):
+        """
+        Add a layer to the end (top) of the group
+
+        :param layer: The layer to add
+        """
+
+        assert layer is not self
+
+        self.add_layers([layer])
+
+        return layer
+
+    def add_layers(self, layers = []):
```

Comment:
Default argument should not be a mutable object. In this case drop the default arg

Reply:
I had left that because the layers variable itself isn't being written into when the default case occurs but true i should have removed it.